### PR TITLE
feat: prepare for cosmos claims wire-up

### PR DESCRIPTION
**Pull request by gomesalexandre**
## Description

This lifts some state up, removes hardcoded data and brings it to the components from selectors so that we can fire the claim Tx.

## Notice

- [x] Have you followed the guidelines in our [Contributing]('https://github.com/shapeshift/web/CONTRIBUTING.md) guide?
- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/shapeshift/web/pulls) for the same update/change?

## Pull Request Type

- [ ] :bug: Bug fix (Non-breaking Change: Fixes an issue)
- [ ] :hammer_and_wrench: Chore (Non-breaking Change: Doc updates, pkg upgrades, typos, etc..)
- [x] :nail_care: New Feature (Breaking/Non-breaking Change)

## Issue (if applicable)

#670 

## Risk

N/A - Effectively, this just lifts some logic up and uses selector data instead of hardcoded data, there is no real business logic change.

## Testing

The cosmos staking flow doesn't regress.

### CODE REVIEW

File: src/plugins/cosmos/components/modals/Staking/Staking.tsx
Comment (0xdef1cafe) [inline src/plugins/cosmos/components/modals/Staking/Staking.tsx:129]: get this from global state

**Review by 0xdef1cafe — CHANGES_REQUESTED** [review]

File: src/plugins/cosmos/components/modals/Staking/Staking.tsx
Comment (gomesalexandre) [reply src/plugins/cosmos/components/modals/Staking/Staking.tsx]: New selector added ✅

**Review by elmutt — APPROVED** [review]

File: src/plugins/cosmos/components/modals/Staking/views/ClaimConfirm.tsx
Comment (DaoDev44) [inline src/plugins/cosmos/components/modals/Staking/views/ClaimConfirm.tsx:47]: Nitpick for sure, but wonder if this should live in `lib`?

Comment (gomesalexandre) [reply src/plugins/cosmos/components/modals/Staking/views/ClaimConfirm.tsx:47]: You can expect it will go away either before, or right after release, I'm still on the hunt for all hardcoded things and the ShapeShift validator address can be surfaced from the validator slice in https://github.com/shapeshift/web/pull/1356

**Review by DaoDev44 — APPROVED** [review]

File: src/plugins/cosmos/components/modals/Staking/Staking.tsx
Comment (elmutt) [inline src/plugins/cosmos/components/modals/Staking/Staking.tsx:93]: I dont think we should be calling this accountSpecifier here or in any of the selectors. Its not the same accountSpecifier we use other places and makes it confusing

File: src/state/slices/accountSpecifiersSlice/selectors.ts
Comment (elmutt) [inline src/state/slices/accountSpecifiersSlice/selectors.ts:14]: Might be nice to have a bit more comments describing why this is needed and how used. Its not super clear based on the name

File: src/plugins/cosmos/components/modals/Staking/Staking.tsx
Comment (gomesalexandre) [reply src/plugins/cosmos/components/modals/Staking/Staking.tsx:93]: Let's rename it after release after we agree on new terminology and standard format? That would require renaming it in many places, in code that's in develop as well as code that's on various PRs currently so it's a bit tricky to tackle it all at once while we have PRs opened

Comment (elmutt) [reply src/plugins/cosmos/components/modals/Staking/Staking.tsx:93]: totally agree. I just wanted to make a note of it here so we dont forget

File: src/state/slices/accountSpecifiersSlice/selectors.ts
Comment (0xdef1cafe) [inline src/state/slices/accountSpecifiersSlice/selectors.ts:17]: will defer to @cjthompson here - let's all this `selectAccountSpecifier` now until we land on something better, clearly `pubkeyish` flopped

**Review by 0xdef1cafe — DISMISSED** [review]
one small requested change as to not make this whole accountSpecifier/pubkeyish mess worse, jointly bad idea from both of us sorry

File: src/state/slices/accountSpecifiersSlice/selectors.ts
Comment (gomesalexandre) [reply src/state/slices/accountSpecifiersSlice/selectors.ts]: Renamed

Comment (gomesalexandre) [reply src/state/slices/accountSpecifiersSlice/selectors.ts]: Do we also want to rename references to `pubkeyish` to `accountSpecifier` in the body/comments? IMO we should keep them as it makes it quite clear that it's not necessarily an `accountSpecifier` (until we have some more discussions about this terminology and format, cc. @elmutt @DaoDev44)

File: src/plugins/cosmos/components/modals/Staking/views/ClaimBroadcast.tsx
Comment (cjthompson) [inline src/plugins/cosmos/components/modals/Staking/views/ClaimBroadcast.tsx:5]:
```suggestion
import { bnOrZero } from 'lib/bignumber/bignumber'
```

**Review by cjthompson — APPROVED** [review]

**Pull request merged**
